src/api/sessionApi.js

```diff
@@ -12,10 +12,22 @@ class Session {
   static logout() {
     return api.delete('/users/sign_out');
   }
-
   static signUp(user) {
     return api.post('/users', user);
   }
+  static updateUser(user) {
+    const { id } = user;
+    return api.put(`users/${id}`, { user });
+  }
+  static getTopics() {
+    return api.get('topics');
+  }
+  static createTarget(target) {
+    return api.post('targets', { target });
+  }
+  static getTargets() {
```

Comment:
Same here. This doesn't look like a session api endpoint. Can you move it to another file? Maybe you should have targetApi and topicApi. Same with actions/reducers. It's nice to keep concerns separate.